app/views/checkout/payment.scala.html

```diff
@@ -157,7 +158,11 @@
                                     Delivery address
                                 </legend>
                                 <div class="field-panel__fields">
-                                    @fragments.checkout.fieldsPaper(productData, countriesWithCurrency)
+                                    @if(productData.plans.list.contains(Delivery)) {
```

Comment:
@tomverran Looks like this is actually looking through all the products, not just the one currently displayed. That means it will always default to the paper fields.

I'm trying to figure out the correct way of seeing whether we're serving the voucher checkout page without resorting to looking at the slug.